docs/user-guide/release-notes/2.0.x.rst

```diff
@@ -1,6 +1,19 @@
 2.0 Release Notes
 =================
 
+2.0.1
```

Comment:
Ok thanks, I'll get that updated. FWIW I don't think it was a redmine problem, but rather that 8 days ago when I submitted this, 2.0.1 was not yet releasing. ;)